eval/eval_test.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"grol.io/grol/ast"
 	"grol.io/grol/eval"
+	_ "grol.io/grol/extensions"
```

Comment:
I'm not fan of this. It uses init and blank import.

https://github.com/leighmcculloch/gochecknoinits

https://peter.bourgon.org/blog/2017/06/09/theory-of-modern-go.html

https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#blank-imports


I would expect the code in this file to call a register method in extensions

Reply:
yeah the init() is an issue... not sure how to do extensions some other way though, well I guess I could call extensions.Init() yes

will do that

Reply:
done in https://github.com/grol-io/grol/pull/96/commits/2fc25bc43e98d1d01ae13225bc8a61396922a857 ptal